### DISCUSSION TITLE
weston 如何配置拼音输入法

### DISCUSSION THREAD

**Discussion by guyuming76** [fcitx/fcitx5#665] [Q&A] [answered]
我在gentoo上安装了weston,可以启动运行，另外用 fcitx5 -d --verbose "*=5"  命令在一个VT里面启动了fcitx5, 但在另一各VT（alacritty 或 foot）里，按ctrl+space 并不能切换到中文输入法，输入法的popup不显示。

![wayland-screenshot-2022-11-25_11-14-58](https://user-images.githubusercontent.com/12045548/203893287-0769b6c2-a3d5-48a4-b270-f4814223b8ca.png)

**Comment by wengxt** [comment, marked as answer]
weston不支持text input v3
